src/Hazelcast.Net.Linq.Async/Naming.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+// http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+
+namespace Hazelcast.Linq
+{
+    internal class Naming
+    {
+        public static string EscapeColumnName(string s)
+        {
+            return s.Equals("value", StringComparison.OrdinalIgnoreCase)
```

Comment:
If we escape that, user also should create the mapping accordingly which is like you did in the tests. I'm still not sure whether to touch naming since user should also participate to make it work but could you at least mention name escaping at LINQ docs?

Reply:
Valid concern. I did it because it was messing with tests, as you mention, but now ... yes ... it can be confusing. The situation "before" was that if I have a `HMap<string, Thing>` and `Thing` has a `Value` property, any LINQ query such as `.Where(x => x.Value == 2)` would cause an SQL exception. With this change... it works with the adjusted mapping, else it also produces a SQL exception.

Reply:
Have removed the magic thing and will note it as an issue.

Reply:
Thanks. We can provide extensions for custom naming later.